Demo/Controllers/LoginViewController.swift

```diff
@@ -14,13 +14,31 @@ class LoginViewController: UIViewController {
     @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
     @IBOutlet weak var usernameTextField: UITextField!
     
+    // MARK: - Properties
+    
+    private var chatkit: Chatkit?
+    
     // MARK: - View lifecycle
     
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         self.displayKeyboard()
     }
     
+    // MARK: - Navigation
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        super.prepare(for: segue, sender: sender)
+
+        if segue.identifier == LoginViewController.loginSegue {
```

Comment:
Its not technically a change introduced by this PR but do you think we could rename `loginSegue` to `loginSegueId` so its type/value is a little more explicit?

Reply:
Sure, no problem.